tests/unit/database_tests.py

```diff
@@ -1479,25 +1460,17 @@ def test_get_search_result_executes_search_query_with_group_option(self):
         )
         # for group parameter options, 'rows' results are within 'groups' key
         self.assertEqual(len(resp['groups']), 5)
+
+        results = []
+        for group in resp['groups']:
+            results.append(group['by'])
+            self.assertEqual(1, group['total_rows'])
+            self.assertEqual(1, len(group['rows']))
+            self.assertEqual('julia', group['rows'][0]['fields']['name'])
+
         self.assertEqual(
-            resp,
-            {'total_rows': 100, 'groups': [
```

Comment:
We were asserting the contents of `rows` before, why not continue that in the for loop?